src/components/UI/Slider/SliderWithMarkers.vue

```diff
@@ -41,6 +41,11 @@ export default {
     };
   },
   created() {
+    // set marker color to orange if plio is published else set it to
+    // default
+    if(this.isDragDisabled) this.setMarkerColor('#F78000')
```

Comment:
is automatic linting not on for you?
```suggestion
    if (this.isDragDisabled) this.setMarkerColor('#F78000')
```